tests/test_helper.py

```diff
@@ -17,3 +19,42 @@ def test_numpy_softmax(shape, axis):
     np_softmax = numpy_softmax(logits, axis=axis)
     torch_softmax = torch.from_numpy(logits).softmax(dim=axis).numpy()
     np.testing.assert_array_almost_equal(np_softmax, torch_softmax)
+
+
+@pytest.mark.parametrize(
+    'inputs',
+    [
+        (
+            DocumentArray(
+                [
+                    Document(text='hello, world'),
+                    Document(text='goodbye, world'),
+                    Document(
+                        text='hello, world',
+                        uri='https://docarray.jina.ai/_static/favicon.png',
```

Comment:
also coverage the docs with `blob` and `tensor`

Reply:
`tensor = [0, 1, 2, 3]` or `tensor = np.array([0, 1, 2, 3])`

Reply:
🥹